tedana/reporting/html_report.py

```diff
@@ -121,8 +121,8 @@ def generate_report(out_dir, tr):
         df._link_figures(fig, comptable_cds, div_content, out_dir=out_dir)
 
     # Create a layout
-    app = layouts.column(layouts.row(kappa_rho_plot, kappa_sorted_plot,
-                                     rho_sorted_plot, varexp_pie_plot),
+    app = layouts.column(layouts.row(kappa_rho_plot, kappa_sorted_plot),
+                         layouts.row(rho_sorted_plot, varexp_pie_plot),
                          div_content)
```

Comment:
We probably should patch up this `div_content`, too. 
We should make a second `layouts.column` if we want to make sure space is preserved for the image... We'll also need to adjust the dimensions of `div_content` to match. Probably something more like `width=500, height=750`

Reply:
And, as a very minor, totally ignorable opinion, I'd put `kappa_sorted_plot` and `rho_sorted_plot` in the same row, personally !

Reply:
I'm not sure I understand what you mean by making a second layout.column... Do you want to have two columns instead of two rows?